public/app/features/storage/FolderDashboard.tsx

```diff
@@ -0,0 +1,69 @@
+import { css } from '@emotion/css';
+import React from 'react';
+import { useAsync } from 'react-use';
+
+import { DataFrame, GrafanaTheme2 } from '@grafana/data';
+import { Card, Icon, useStyles2 } from '@grafana/ui';
+
+import { DashboardModel } from '../dashboard/state';
+
+import { getGrafanaStorage } from './storage';
+
+interface Props {
+  dash: DashboardModel;
+}
+
+export function FolderDashboard({ dash }: Props) {
```

Comment:
Yes, this should eventually be the same as https://github.com/grafana/grafana/blob/83a66d46e194684dcf103c2fa3d2cd8d16113041/public/app/features/search/page/components/FolderView.tsx#L70